cl/audio/feeds.py

```diff
@@ -30,20 +32,26 @@ class JurisdictionPodcast(JurisdictionFeed):
     item_enclosure_mime_type = "audio/mpeg"
 
     def title(self, obj):
-        return f"Oral Arguments for the {obj.full_name}"
+        court = obj[1]
+        return f"Oral Arguments for the {court.full_name}"
+
+    def get_object(self, request, court):
+        return request, get_object_or_404(Court, pk=court)
 
     def items(self, obj):
         """
         Returns a list of items to publish in this feed.
         """
-        if waffle.flag_is_active(obj, "oa-es-deactivate"):
+        request = obj[0]
+        court = obj[1]
```

Comment:
I think we can refactor these lines using tuple unpacking:

```suggestion
        request, court = obj
```

Reply:
Yeah, this is simpler. Changed.